### PR TITLE
WIP: Fix jaas integration test

### PR DESCRIPTION
**Pull request by hmlanigan**
## Description

Update how microk8s.config is provided to the tests to allow the TestAcc_ResourceKubernetesCloud to run.

## Type of change

- Github action updated

## QA steps

In the Integration-JAAS action, TestAcc_ResourceKubernetesCloud should not fail due to lack of micrko8s config.

### CODE REVIEW

File: .github/workflows/test_integration_jaas.yaml [hunk 1]
```diff
@@ -71,9 +71,6 @@ jobs:
           sudo microk8s.enable dns storage
           sudo microk8s.enable dns local-storage
```
Comment (anvial) [inline .github/workflows/test_integration_jaas.yaml:72]: Let's change this to:

```
sudo microk8s enable hostpath-storage
sudo microk8s enable dns
```

File: .github/workflows/test_integration_jaas.yaml [hunk 2]
```diff
@@ -93,6 +90,12 @@ jobs:
           echo "EOF" >> $GITHUB_ENV
           echo "TEST_MANAGEMENT_BR=10.150.40.0/24" >> $GITHUB_ENV
           echo "TEST_PUBLIC_BR=10.170.80.0/24" >> $GITHUB_ENV
+      - name: "Set additional environment for LXD"
+        # Setup the microk8s config to be used by the tests.
+        if: ${{ matrix.action-operator.cloud == 'lxd' }}
+        # language=bash
+        run: |
+          sudo microk8s.config > /home/$USER/microk8s-config.yaml
```
Comment (anvial) [inline .github/workflows/test_integration_jaas.yaml:98]: Something wrong with StdOut forwarding here. Try:

```
sudo microk8s.config | tee /home/$USER/microk8s-config.yaml
```
that should help

**Comment by hmlanigan** [comment]
this work is getting picked up in #641

**Pull request closed without merging**